docs/description/clang-analyzer-deadcode.md

```diff
@@ -0,0 +1,9 @@
+.. meta:: :http-equiv=refresh:
+5;URL=https://clang.llvm.org/docs/analyzer/checkers.html\#deadcode-deadstores
+
+clang-analyzer-deadcode.DeadStores
+==================================
+
+The clang-analyzer-deadcode.DeadStores check is an alias, please see
+`Clang Static Analyzer Available Checkers <https://clang.llvm.org/docs/analyzer/checkers.html#deadcode-deadstores>`\_
```

Comment:
This is not a markdown link, it would be nice to have it as a clickable link

Reply:
Made all links clickable ✅